static/base.css

```diff
@@ -543,6 +543,10 @@ color: #00bb00;}
     font-size: 16px;
 }
 
+.parameterTable .tableAddNewConfigBtn, .eagleTableDisplay .tableAddNewConfigBtn{
```

Comment:
Another place in the CSS (.rightWindow .btn-primary) you override both the background-color and the border-color.

Maybe you want to do that here? Or combine these same CSS rules.

Reply:
good catch, ive combined the rules